File: matlab/q009.m
% There exists exactly one Pythagorean triplet for 
% which a + b + c = 1000.
% Find the product abc.
% a < b < c
% answer = 31875000

for a=1:600
    for b=1:600
        c=sqrt(a.^2+b.^2);
        if a+b+c==1000;
            a1=a;
            b1=b;
            c1=c;
            break
        end
    end
end

a1*b1*c1

File: java/src/q009.java
/*
There exists exactly one Pythagorean triplet for
which a + b + c = 1000.
Find the product abc.
a < b < c
answer = 31875000
 */

package src;

public final class q009 implements Euler {

    public String solve(){
        double c = 0;
        for (double b=1;b<601;b++){
            for (double a=1;a<601;a++){
                c = Math.sqrt((Math.pow(a,2)) + (Math.pow(b,2)));
                if (a + b + c == 1000){
                    return String.format("%.0f",a*b*c);
                }
            }
        }
        return "";
    }


}
